bin/grommet.js

```diff
@@ -17,76 +18,120 @@ String.prototype.capitalize = function() {
   return words.join(' ');
 };
 
-function nodeVersionSupported() {
-  return Number(process.version.match(/^v(\d+\.\d+)/)[1]) >= 0.10;
+function fileExists(filePath) {
+  try {
+      result = fs.statSync(filePath);
```

Comment:
can we keep the two-character spacing in here? I believe we have 4, where it is supposed to be two.

I need to add the jslint in this file as well. so my apologies.